src/runtime/ltypes/ltypes.py

```diff
@@ -51,3 +51,31 @@ def __call__(self, *args, **kwargs):
 def overload(f):
     overloaded_f = OverloadedFunction(f)
     return overloaded_f
+
+
+
+# C interoperation support
+
+class CTypes:
+    """
+    A wrapper class for interfacing C via ctypes.
+    """
+
+    def __init__(self, f):
+        self.name = f.__name__
+        self.args = f.__code__.co_varnames
+        self.annotations = f.__annotations__
```

Comment:
We can skip them if they aren't used anywhere in the class?

Reply:
They are used now.